apps/ehr/tests/e2e-utils/resource-handler.ts

```diff
@@ -198,31 +194,19 @@ export class ResourceHandler {
       }
 
       // Create appointment and related resources using zambda
-      const appointmentData =
-        this.#flow === 'in-person'
-          ? await createSamplePrebookAppointments({
-              oystehr: this.#apiClient,
-              authToken: getAccessToken(),
-              phoneNumber: formatPhoneNumber(PATIENT_PHONE_NUMBER)!,
-              createAppointmentZambdaId: this.#createAppointmentZambdaId,
-              zambdaUrl: process.env.PROJECT_API_ZAMBDA_URL,
-              selectedLocationId: inputParams?.selectedLocationId ?? process.env.LOCATION_ID,
-              demoData: patientData,
-              projectId: process.env.PROJECT_ID!,
-              paperworkAnswers: this.#paperworkAnswers,
-            })
-          : await createSampleTelemedAppointments({
-              oystehr: this.#apiClient,
-              authToken: getAccessToken(),
-              phoneNumber: formatPhoneNumber(PATIENT_PHONE_NUMBER)!,
-              createAppointmentZambdaId: this.#createAppointmentZambdaId,
-              islocal: process.env.APP_IS_LOCAL === 'true',
-              zambdaUrl: process.env.PROJECT_API_ZAMBDA_URL,
-              locationState: inputParams?.telemedLocationState ?? process.env.STATE_ONE, // todo: check why state is used here
-              demoData: patientData,
-              projectId: process.env.PROJECT_ID!,
-              paperworkAnswers: this.#paperworkAnswers,
-            });
+      const appointmentData = await createSampleAppointments({
+        oystehr: this.#apiClient,
+        authToken: getAccessToken(),
+        phoneNumber: formatPhoneNumber(PATIENT_PHONE_NUMBER)!,
+        createAppointmentZambdaId: this.#createAppointmentZambdaId,
+        zambdaUrl: process.env.PROJECT_API_ZAMBDA_URL,
+        serviceMode: this.#flow === 'telemed' ? ServiceMode.virtual : ServiceMode['in-person'],
+        selectedLocationId: inputParams?.selectedLocationId ?? process.env.LOCATION_ID,
+        locationState: inputParams?.telemedLocationState ?? process.env.STATE_ONE, // todo: check why state is used here
```

Comment:
```suggestion
        locationState: inputParams?.telemedLocationState ?? process.env.STATE_ONE,
```

this process.env.STATE_ONE is used to give us a consistent state for our automated test suite right?